containers/ecr-viewer/src/app/services/accordionItemService.tsx

```diff
@@ -17,31 +17,23 @@ import {
   evaluateProviderData,
   evaluateFacilityData,
 } from "@/app/services/evaluateFhirDataService";
-import { evaluateClinicalData } from "./common";
-import { Accordion } from "@trussworks/react-uswds";
+import { evaluateClinicalData } from "../view-data/components/common";
+import { Accordion, HeadingLevel, Tag } from "@trussworks/react-uswds";
```

Comment:
q: this "service" uses _a lot_ of components, but it doesn't return a component - feels a bit off

Reply:
made this a helper file next to the `EcrDocument` component in its shiny new folder